JS SDK clarification re: IP address (#182)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/README.md

);
```

In the above example, information such as the `userId` and `email` along with the [**contextual information**](https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/common-fields#javascript-sdk) is captured.

{% hint style="info" %}
Note that the user's IP address is not automatically collected by the SDK. Instead, it is filled in by the RudderStack server when it receives the client-side events.

> {% endhint %}

Reviewer: This is standard practice for capturing IP addresses. Let's not mention this here.

{% hint style="warning" %}

> If you explicitly specify the IP address in the event, RudderStack will use that IP instead of automatically capturing it when collecting the events. You can use this feature to anonymize your users' IP - e.g., by supplying a random dummy IP address.

Reviewer: ```suggestion
If you explicitly specify the IP address in the event, RudderStack will use that IP instead of capturing it on the backend. You can use this feature to anonymize your users' IP - e.g., by supplying an anonymous IP address.
```
